app/experimenter/experiments/forms.py

```diff
@@ -552,7 +552,22 @@ def clean(self):
         return cleaned_data
 
 
+class RadioWidget(forms.widgets.RadioSelect):
+    template_name = "experiments/radio_widget.html"
+
+
+class RadioWidgetCloser(forms.widgets.RadioSelect):
+    template_name = "experiments/radio_widget_closer.html"
+
```

Comment:
I make a custom widget because the `RadioWidget` created here and used in the Risks section uses col-2 for spacing of the Yes and No buttons. This spacing is too wide for this application, and col-1s are too small for the button and label. I take them out of columns and just use some padding.

Reply:
I think this class could use a docstring saying something along these lines.